docs/topics/Gateway.md

```diff
@@ -833,6 +846,45 @@ The inner `d` key is a boolean that indicates whether the session may be resumab
 
 `APPLICATION_COMMAND_PERMISSIONS_UPDATE` event, sent when an application command's permissions are updated. The inner payload is an [application command permissions](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-permissions-object-guild-application-command-permissions-structure) object.
 
+### Auto Moderation
+
+All auto moderation related events are currently only sent to bot users which have the `MANAGE_GUILD` permission.
```

Comment:
Will auto moderation event payloads also have a general `id` field? A majority of other Gateway events do this

Reply:
not sure i'm understanding, what would the general `id` field be used to uniquely identify here? the events related to automod rule config changes will have an `id` field set to the rule id if that's what you're looking for.